wallet/src/components/Offer.tsx

```diff
@@ -141,11 +134,14 @@ const OfferWithoutContext = ({
         color={statusColors[status]}
         label={statusText[status]}
       />
-      <span className="Date text-gray">{formatDateNow(date)}</span>
       <div className="OfferOrigin" style={{ wordBreak: 'break-word' }}>
         <PetnameSpan name={sourceDescription} />
-        <i> via </i>
-        <span className="Blue">{dappOrigin || origin}</span>
+        {(dappOrigin || origin) && (
```

Comment:
consider `const actualOrigin = dappOrigin || origin` so the condition doesn't need to be repeated on line 142

Reply:
Nice done.